snape/make_dataset.py

```diff
@@ -388,6 +391,8 @@ def _safe_get_with_default(cfg, key, default):
     tail_strength = _safe_get_with_default(config, 'tail_strength', 0.5)
     noise = _safe_get_with_default(config, 'noise', 0.)
     seed = _safe_get_with_default(config, 'random_seed', 42)
+    shuffle = _safe_get_with_default(config, 'shuffle', "Yes")
+    shuffle = True if shuffle == "Yes" else False
```

Comment:
This seems brittle. With this logic, even `"yes"` would evaluate to `False`. I think it makes more sense to pass a bool directly in the config rather than a string. Thoughts?

Reply:
Yes. That would be much simpler.
A similar issue exists for the `entropy` and `star_schema` config options.